reactor-core/src/main/java/reactor/core/scheduler/ElasticScheduler.java

```diff
@@ -87,11 +87,8 @@
 		this.factory = factory;
 		this.cache = new ConcurrentLinkedDeque<>();
 		this.all = new ConcurrentLinkedQueue<>();
-		this.evictor = Executors.newScheduledThreadPool(1, EVICTOR_FACTORY);
-		this.evictor.scheduleAtFixedRate(this::eviction,
-				ttlSeconds,
-				ttlSeconds,
-				TimeUnit.SECONDS);
+		//evictor is now started in `start()`. make it look like it is constructed shutdown
+		this.shutdown = true;
```

Comment:
WDYT about using an inverse, `started`? Since there is no `shutdown` action, and we use this flag to protect from the double start.

Reply:
`dispose()` is the shutdown action. now for `ElasticScheduler`, when `dispose()` is called then tasks will be rejected. as soon as `start()` is called (for the second time), tasks can be re-submitted

Reply:
oh, sorry, I somehow assumed that `shutdown` is a newly added field 🤦